### PR TITLE
Molgenis filebeat

### PR DESCRIPTION
**Pull request by akuperus**
#### Checklist
- [x] Functionality works & meets specifications (tested on rancher)
- [x] Templates reviewed
- [x] Bumped the chart version
- [x] Updated appVersion (if needed)

### CODE REVIEW

File: charts/molgenis-filebeat/.helmignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+tests/
+.pytest_cache/
```
Comment (christianhilbrands) [inline charts/molgenis-filebeat/.helmignore]: could you remove .helmignore?

File: charts/molgenis-filebeat/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+description: Official Elastic helm chart for Filebeat
+home: https://github.com/elastic/helm-charts
+maintainers:
+- email: helm-charts@elastic.co
+  name: Elastic
+name: filebeat
+version: 7.5.2
+appVersion: 1.0.0
```
Comment (christianhilbrands) [inline charts/molgenis-filebeat/Chart.yaml]: appVersion and version are swopped. Could you change it?

File: charts/molgenis-filebeat/Makefile [hunk 3]
```diff
@@ -0,0 +1 @@
+include ../helpers/common.mk
```
Comment (christianhilbrands) [inline charts/molgenis-filebeat/Makefile]: Is not needed i think?

File: charts/molgenis-filebeat/tests/filebeat_test.py [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+        == name
+    )
+    volumes = r["daemonset"][custom_name]["spec"]["template"]["spec"]["volumes"]
+    assert {
+        "name": "data",
+        "hostPath": {
+            "path": "/var/lib/" + custom_name + "-default-data",
+            "type": "DirectoryOrCreate",
+        },
+    } in volumes
```
Comment (christianhilbrands) [inline charts/molgenis-filebeat/tests/filebeat_test.py]: Does this python script need to be added to the helm chart?

File: charts/molgenis-filebeat/values.yaml [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+# This is the PriorityClass settings as defined in
+# https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+priorityClassName: ""
+
+updateStrategy: RollingUpdate
+
+# Override various naming aspects of this chart
+# Only edit these if you know what you're doing
+nameOverride: ""
+fullnameOverride: ""
```
Comment (christianhilbrands) [inline charts/molgenis-filebeat/values.yaml]: i think you have forked the repo? What we are doing is extending on top of the existing repo. I am not sure if this is the proper way to go.

File: charts/molgenis-filebeat/Chart.yaml (hunk 2 above)
Comment (akuperus) [reply charts/molgenis-filebeat/Chart.yaml]: done

File: charts/molgenis-filebeat/.helmignore (hunk 1 above)
Comment (akuperus) [reply charts/molgenis-filebeat/.helmignore]: done

File: charts/molgenis-filebeat/Makefile (hunk 3 above)
Comment (akuperus) [reply charts/molgenis-filebeat/Makefile]: removed

File: charts/molgenis-filebeat/tests/filebeat_test.py (hunk 4 above)
Comment (akuperus) [reply charts/molgenis-filebeat/tests/filebeat_test.py]: removed

File: charts/molgenis-filebeat/values.yaml (hunk 5 above)
Comment (akuperus) [reply charts/molgenis-filebeat/values.yaml]: fixed

File: charts/filebeat/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+description: Official Elastic helm chart for Filebeat
+home: https://github.com/elastic/helm-charts
+maintainers:
+- email: helm-charts@elastic.co
+  name: Elastic
```
Comment (christianhilbrands) [inline charts/filebeat/Chart.yaml]: i should put your own name here because you made changes to it

**Pull request merged**